Update version numbers to match PyPI

bsipocz opened the pull request:
@mwcraig - I guess the only problematic point is `gwcs` that just got on PyPI very recently. All the others are trivial updates of the version numbers to match the latest releases on PyPI (with the exception of ginga, where I wasn't sure it's worth updating to the ever changing newest).

bsipocz committed:
Update astropy-helpers version

bsipocz committed:
Update astroquery version

bsipocz committed:
Update pydl version

bsipocz committed:
Adding back gwcs as 0.5 now seems to be on pypi

bsipocz committed:
Adding pyasdf as it's a depencency of gwcs

bsipocz committed:
hacking to add numpy dependency to pyasdf that accidentally doesn't list it in it's setup.py

mwcraig commented:
@bsipocz -- thanks very much for this! A recipe template will need to be created for asdf to handle the numpy dependency. I can do a PR on your PR to do that tonight or you can create it if you want. 

I usually generate a recipe template with `conda skeleton pypi --setup-args="--offline" asdf` 

~~The only hassle here is that you need to use the version of conda-build I've bundled as a submodule because my PR to add the `--setup-args` option has not been merged (yet).~~

Actually, it is easier than I thought: if you have `conda-affiliate-builder` cloned on your machine, then `python affiliate_builder/prepare_packages.py requirements.yml` should generate recipes for you, which you could then use to make a recipe template.

mwcraig commented on requirements.yml:
```diff
@@ -108,11 +108,15 @@
+- name: asdf
+  version: 1.0.0
+  astropy_helpers: true
+  numpy_compiled_extensions: yes
```
`numpy_compiled_extensions` should be `false` for `asdf` because it is a pure python package.

bsipocz replied:
yes sorry, I was just a desperate to try to make it work (and don't know much about how the package building is working).

mwcraig committed:
Add asdf recipe template

mwcraig commented:
@bsipocz -- this _should_ build once you merge the PR I just made. Do you want to do the asdf fix to `install_requires` or shall I?

bsipocz committed:
Merge pull request #1 from mwcraig/add-asdf-recipe

Add asdf recipe template

bsipocz commented:
@mwcraig - Thanks. I already opened a PR for pyasdf https://github.com/spacetelescope/pyasdf/pull/185, and this should be fixed in its next release. Should I open an issue as a reminder that the recipe can be removed with the new release?

mwcraig commented:
@bsipocz -- yes please!

mwcraig commented:
@bsipocz -- looks like gwcs depends on asdf but doesn't list the requirement in setup.py. Same division of labor as for asdf, if that works for you...(I'll add the recipe template, you patch the original repo/make a reminder issue).

mwcraig committed:
Add recipe template for gwcs

bsipocz commented:
Ohh, great. Sorry for the unplanned messing with these.

bsipocz committed:
Merge pull request #2 from mwcraig/add-gwcs-recipe

Add recipe template for gwcs

bsipocz commented:
Thanks @mwcraig. I vaguely remember that either of them has some version requirement for the latest (1.1) astropy, but probably it will be fine as is now.

mwcraig commented:
Next up: `pydl`. Issue is a default path in a script that relies on the environment variable `HOME` being set...which is not the case on Windows. Just googled the fix, so I'll add the recipe template and do the PR on pydl.

mwcraig committed:
Add recipe temple for pydl

bsipocz committed:
Merge pull request #3 from mwcraig/add-pydl-recipe

Add recipe temple for pydl

mwcraig commented:
@bsipocz -- now an `asdf` failure on python 3, because it tries to import a compatibility package that, I'm guessing, isn't installed on py3k...will come back to this on Saturday.

bsipocz committed:
Update version for asdf

mwcraig committed:
Skip import test for _odict_py2

See https://github.com/spacetelescope/pyasdf/issues/187 for a discussion
of why _odict_py2 does not end up being included in python 3 source
packages in asdf.

bsipocz committed:
Merge pull request #4 from mwcraig/skip-one-more-asdf-test

Skip import test for _odict_py2

bsipocz commented:
Thanks @mwcraig for all your inputs, and I'm really sorry that this PR (that I thought would be totally harmless) required that much work from you.

mwcraig commented:
:tada: :confetti_ball: :balloon: :smile:

No problem, thanks for all of your work on it. The issues we ran into were coming sooner or later, so I'm happy to have gotten them out of the way now!
